### DISCUSSION TITLE
Flag newly added apps in the app store

### DISCUSSION THREAD

**Discussion by Wiz0r** [runtipi/runtipi-appstore#4315] [Ideas]
I really enjoy discovering and trying new apps. It would be very helpful to assign a 'new' flag to completely new app store apps.

One possible approach could be to automatically apply this flag for one month and then remove it.

**Comment by cori** [comment]
I've been wanting a sort or filter of this nature too. A flag would be useful but still requires scrolling through the whole list when you might just want to look at new additions.

**Comment by blq-vdcj** [comment]
I have also asked it in Discord, some QOL will upgrade the Ux

**Comment by JigSawFr** [comment]
Now available ;)

**Reply by Wiz0r** [reply]
Thank you for the implementation. I already saw the first "new" flag! :-)
